popSimManu.tex

```diff
@@ -82,6 +80,8 @@ \section*{Introduction}
 While en masse these methods have increased our understanding of the
 impacts of genetic and evolutionary processes, very little has been done to systematically
 benchmark the quality of inferences gleaned from computational population genetics.
+As large databases of population genetic variation begin to be used to inform public health procedures,
+the accuracy and quality of these inferences is becoming even more important.
```

Comment:
Maybe some citations here? We could say explicitly that e.g. popgen inferences sometimes provide important priors for inference of causal SNPs; this might be important for PRS.